pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue

```diff
@@ -322,6 +322,7 @@ export default defineComponent({
             class="checkbox-info icon icon-info icon-lg"
             :data-testid="componentTestid + '-info-icon'"
             :tabindex="isDisabled ? -1 : 0"
+            role="button"
```

Comment:
Should we be using the tooltip role instead? 

`role="tooltip"`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tooltip_role